project-base/storefront/components/Pages/Contact/contactFormMeta.tsx

```diff
@@ -17,12 +20,14 @@ export const useContactForm = (): [UseFormReturn<ContactFormType>, ContactFormTy
             email: Yup.string().required(t('Please enter email')).email(t('This value is not a valid email')).min(5),
             name: Yup.string().required(t('Please enter your name')),
             message: Yup.string().required(t('Please enter a message')),
+            privacyPolicy: Yup.boolean().isTrue(t('You have to agree with our privacy policy')),
```

Comment:
Looking at this form, maybe the email rule can be taken from `validationRules.ts`. I know this is not a LOC you have changed in this PR, but just so that you leave the code in a better state than the one you found it in :)

Reply:
Thanks for the suggestion, I will use the validation rule so it is consistent across the codebase.